test/framework/alertmanager.go

```diff
@@ -347,10 +355,14 @@ func (f *Framework) GetSilences(ns, n string) (models.GettableSilences, error) {
 // configuration via the Alertmanager's API and checks if it contains the given
 // string.
 func (f *Framework) WaitForAlertmanagerConfigToContainString(ns, amName, expectedString string) error {
+
+	var pollError error
```

Comment:
you'd need to modify L376 to report `pollErr` there (see L254 for example). Also it would make sense to set  `pollErr` when the returned config doesn't match with the expected value.

Reply:
Done...PTAL